app/utility/base_world.py

```diff
@@ -109,6 +111,19 @@ async def walk_file_path(path, target):
                 return os.path.join(root, '%s.xored' % target)
         return None
 
+    @staticmethod
+    def get_version(path='.'):
+        ignore = ['/plugins/', '/.tox/']
```

Comment:
I feel like this should return a tuple-- the version (number), and the hash, separately. we will display the former on the GUI whereas the latter is for validation only

Reply:
ignore